.github/ISSUE_TEMPLATE/2_new-hub.yml

```diff
@@ -115,56 +67,55 @@ body:
       options:
         - GitHub Authentication (e.g., @mygithubhandle)
         - Google OAuth (e.g., myemailaddress@2i2c.org)
+        - CILogon (yy
         - Username / Password via [auth0](https://auth0.com/docs/connections/database)
```

Comment:
```suggestion
        - Username / Password via auth0: https://auth0.com/docs/connections/database
```

The link doesn't render properly in the drop down

<img width="315" alt="Screenshot 2022-02-10 at 10 04 08" src="https://user-images.githubusercontent.com/44771837/153384639-cee3384d-4559-441a-aa07-29550c4168dc.png">